### PR TITLE
Update example files so that tr_loss is not affected by args.gradient…

### PR DESCRIPTION
**Pull request by Mathieu-Prouveur**
Hi developpers!

Fix training loss value : 

* if gradient_accumulation_steps > 1 then the batch loss value(which is a mean) is scaled by a factor 1/args.gradient_accumulation_steps.

To compare it to evaluation loss it is thus necessary to scale it back by multiplying by args.gradient_accumulation_steps (as done in finetuning script)

Another way to fix this would be to replace the lines with tr_loss/nb_tr_steps by tr_loss/global_step. I thought you might want to consider this alternative

### CODE REVIEW

**Comment by Mathieu-Prouveur** [comment]
Hi developpers! 

Another way to fix this would be to replace the lines with `tr_loss/nb_tr_steps` by `tr_loss/global_step`. I thought you might want to consider this alternative

**Comment by thomwolf** [comment]
Hi @Mathieu-Prouveur, thanks for that.
Indeed I think using `tr_loss/global_step` would be more easy to read.
Can you update this?

**Comment by Mathieu-Prouveur** [comment]
Sure, I've just done the update

**Comment by thomwolf** [comment]
Great, thanks!
